example.js

```diff
@@ -7,7 +7,8 @@ const { prompt } = require('./lib');
         {
             type: 'text',
             name: 'twitter',
-            message: `What's your twitter handle?`
+            message: `What's your twitter handle?`,
+            defaulter: '@prompt'
```

Comment:
here you can just add quotes `'default': 'sasa'` for example.

Reply:
No. It should stay consistent